### ISSUE TITLE
Unable to scatter over multiple arrays using a scatterMethod

### ISSUE THREAD

**Issue by drjrm3** [IBMSpectrumComputing/cwlexec#8]
I am trying to scatter over 2 arrays of the same size using `scatterMethod: flat_crossproduct` and CWLEXEC fails with:

```
com.ibm.spectrumcomputing.cwl.model.process.parameter.type.file.CWLFile cannot be cast to java.lang.CharSequence
```

This seems to happen when using any `scatterMethod`. A failed run with debug info is attached.

[CWLFileCastError.tar.gz](https://github.com/IBMSpectrumComputing/cwlexec/files/2069845/CWLFileCastError.tar.gz)

**Comment by PlatformLSF** [comment]
The scatter inputs are two different type (file, string), but Java is strong type, so when we scatter inputs, we should omit its type

**Comment by PlatformLSF** [comment]
Fixed in 50b76ed
Also test with `dotproduct`
